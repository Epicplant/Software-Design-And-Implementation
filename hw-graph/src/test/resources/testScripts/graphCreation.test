# A test to see that a graph instance can be successfully created

# Creates a Graph
CreateGraph newGraph

# Lists all of the children of node n1
ListChildren newGraph n1