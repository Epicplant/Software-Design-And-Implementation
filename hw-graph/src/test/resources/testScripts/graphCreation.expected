# A test to see that a graph instance can be successfully created

# Creates a Graph
created graph newGraph

# Lists all of the children of node n1
the children of n1 in newGraph are: